test/key_test.rb

```diff
@@ -0,0 +1,34 @@
+require './test/test_helper'
+require './lib/key'
+require 'pry'
+
+class KeyTest < Minitest::Test
+
+  def test_it_exists
+    key = Key.new
+
+    assert_instance_of Key, key
+  end
+
+  def test_random_five_digits
+    key = Key.new
+    array_keys = mock('random array of keys')
+    key.stubs(:random_five_digits).returns([array_keys])
+    assert_equal [array_keys], key.random_five_digits
+  end
+
+  def test_hash_of_keys
```

Comment:
Need to improve testing here.  Mocks/stubs is not fulfilling test coverage requirements.